plugin/pkg/admission/storage/persistentvolume/label/admission_test.go

```diff
@@ -92,32 +89,6 @@ func Test_PVLAdmission(t *testing.T) {
 			},
 			err: nil,
 		},
-		{
-			name:      "cloud provider error blocks creation of volume",
```

Comment:
This test checks for a generic admission functionality and should be reworked to a different volume plugin. Preferably vSphere, because it will go away as the last.

Reply:
Done.